src/nvim/drawline.c

```diff
@@ -449,6 +449,16 @@ static void apply_cursorline_highlight(win_T *wp, linenr_T lnum, int *line_attr,
   }
 }
 
+static bool check_mb_utf8(int *c, int *u8cc)
```

Comment:
Nice change! Is it worth marking this `inline`?

Reply:
Yeah good idea.